moveit_experimental/CMakeLists.txt

```diff
@@ -56,11 +57,10 @@ catkin_package(
     pluginlib
   DEPENDS
     Boost
-    Eigen3
```

Comment:
Yeah, the variables being all caps is really an Eigen problem, but I'm guessing they don't want to change that.

As long as something else states this dependency, things should keep working, but it's something to fix.